ci-operator/jobs/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main-presubmits.yaml

```diff
@@ -156,23 +156,23 @@ presubmits:
     - ^main$
     - ^main-
     cluster: build05
-    context: ci/prow/ci-index-numaresources-operator-bundle
+    context: ci/prow/ci-index
```

Comment:
why do need this rename?

Reply:
And the following as well

Reply:
This one auto generated once I removed the `bundle.as`:
 https://github.com/openshift/release/pull/58597/commits/2de0aaea74185f69203b6e51de025cdd4de52bc9#diff-b719e9d20a9992f73ccfc477a317704367093899f38d6915881e68cdb5af322fL51

Reply:
ok, seems no big deal